src/controller/index.ts

```diff
@@ -564,6 +636,18 @@ export default class SimulariumController {
     public setCameraType(ortho: boolean): void {
         this.visGeometry?.setCameraType(ortho);
     }
+
+    public isPlaying(): boolean {
+        return this.visData.isPlaying;
+    }
+
+    public currentPlaybackHead(): number {
+        return this.visData.currentFrameNumber;
+    }
```

Comment:
this isn't used anywhere, and the installing app should get this through the onTimeChange callback. I don't think we should be adding more public functions to the controller unless there is a clear use case for them to try to keep the API easy to understand, same with the function below